llpc/test/shaderdb/OOB_Check_Optimization_lit.frag

```diff
@@ -5,7 +5,9 @@
 /*
 ; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s -enable-scratch-bounds-checks | FileCheck -check-prefix=SHADERTEST %s
 ; SHADERTEST-LABEL: {{^// LLPC}} final pipeline module info
+; SHADERTEST: define dllexport amdgpu_ps { <4 x float> } @_amdgpu_ps_main
 ; SHADERTEST-NOT: phi
+; SHADERTEST: ret
```

Comment:
This seems unrelated to this patch. Does this test fail without those changes? If not, I think it should be submitted separately.